src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,30 @@
+import { Status } from '../../types/types';
+import { useTodos } from '../../utils/TodoContext';
+import { TodoItem } from '../TodoItem';
+
+export const TodoList: React.FC = () => {
+  const { todos, status } = useTodos();
+
+  let visibleTodos = [...todos];
+
+  if (status !== Status.ALL) {
+    switch (status) {
+      case Status.ACTIVE:
+        visibleTodos = todos.filter(todo => !todo.completed);
+        break;
+      case Status.COMPLETED:
+        visibleTodos = todos.filter(todo => todo.completed);
+        break;
+      default:
+        break;
+    }
+  }
```

Comment:
- Create a helper for filtering and move it from the component
- And `TodoList` shouldn't have logic regarding filtering, it should accept todos via props and render them